content/ratatui-0.23.0.md

```diff
@@ -0,0 +1,276 @@
+
+![horizontal bars](https://user-images.githubusercontent.com/15311024/263077132-ff80b2c4-9d2d-480b-9a72-f3779688bec8.png)
+
+---
+
+### Voluntary skipping capability for Sixel
+
+> [Sixel](https://en.wikipedia.org/wiki/Sixel) is a bitmap graphics format supported by terminals.
+> "Sixel mode" is entered by sending the sequence `ESC+Pq`.
+> The "String Terminator" sequence `ESC+\` exits the mode.
+
+`Cell` widget now has a `set_skip` method that allows the cell to be skipped when copying (diffing) the buffer to the screen. This is helpful when it is necessary to prevent the buffer from overwriting a cell that is covered by an image from some terminal graphics protocol such as Sixel, iTerm, Kitty, etc.
+
+See the pull request for more information: [https://github.com/ratatui-org/ratatui/pull/215](https://github.com/ratatui-org/ratatui/pull/215)
+
+In this context, there is also an experimental image rendering crate: [_ratatu-image_](https://github.com/benjajaja/ratatu-image)
+
+![ratatu-image](https://raw.githubusercontent.com/benjajaja/ratatu-image/master/assets/Recording.gif)
+
+---
+
+### Table/List: Highlight spacing
+
+We added a new property called `HightlightSpacing` to the `Table` and `List` widgets and it can be optionally set via calling `hightlight_spacing` function.
+
+Before this option was available, selecting a row in the table when no row was selected previously made the tables layout change (the same applies to unselecting) by adding the width of the "highlight symbol" in the front of the first column. The idea is that we want this behaviour to be configurable with this newly added option.
+
+```rs
+let list = List::new(items)
+    .highlight_symbol(">>")
+    .highlight_spacing(HighlightSpacing::Always);
+```
+
+Right now, there are 3 variants:
+
+- `Always`: Always add spacing for the selection symbol column.
+- `WhenSelected`: Only add spacing for the selection symbol column if a row is selected.
+- `Never`: Never add spacing to the selection symbol column, regardless of whether something is selected or not.
+
+---
+
+### Table: support line alignment
+
+```rs
+let table = Table::new(vec![
+        Row::new(vec![Line::from("Left").alignment(Alignment::Left)]),
+        Row::new(vec![Line::from("Center").alignment(Alignment::Center)]),
+        Row::new(vec![Line::from("Right").alignment(Alignment::Right)]),
+    ])
+    .widths(&[Constraint::Percentage(100)]);
+```
+
+Now results in:
+
+```text
+Left
+       Center
+               Right
+```
+
+---
+
+### Scrollbar: optional track symbol
+
+The track symbol in the `Scrollbar` is now optional, simplifying composition with other widgets. It also makes it easier to use the `Scrollbar` in tandem with a block with special block characters.
+
+One breaking change is that `track_symbol` needs to be set in the following way now:
+
+```rs
+let scrollbar = Scrollbar::default().track_symbol(Some("-"));
+```
```

Comment:
Can you make a diff between the previous version and this?

Reply:
Sure, done